diff --git a/node_modules/react-native-maps/lib/android/build.gradle b/node_modules/react-native-maps/lib/android/build.gradle
index acc7f57..e4622a0 100644
--- a/node_modules/react-native-maps/lib/android/build.gradle
+++ b/node_modules/react-native-maps/lib/android/build.gradle
@@ -43,7 +43,9 @@ dependencies {
   def googlePlayServicesVersion = rootProject.hasProperty('googlePlayServicesVersion')  ? rootProject.googlePlayServicesVersion : DEFAULT_GOOGLE_PLAY_SERVICES_VERSION
   def androidMapsUtilsVersion   = rootProject.hasProperty('androidMapsUtilsVersion')    ? rootProject.androidMapsUtilsVersion   : DEFAULT_ANDROID_MAPS_UTILS_VERSION
 
-  compileOnly "com.facebook.react:react-native:+"
+  compileOnly('com.facebook.react:react-native:+') {
+    exclude group: 'com.android.support'
+  }
   implementation "com.google.android.gms:play-services-base:$googlePlayServicesVersion"
   implementation "com.google.android.gms:play-services-maps:$googlePlayServicesVersion"
   implementation "com.google.maps.android:android-maps-utils:$androidMapsUtilsVersion"
